tf_files/aws/commons/kube.tf

```diff
@@ -208,6 +207,50 @@ resource "aws_s3_bucket_public_access_block" "kube_bucket_privacy" {
   restrict_public_buckets     = true
 }
 
+resource "aws_s3_bucket" "dbgap_backup_bucket" {
```

Comment:
I'm thinking that rather than setup the bucket and AWS role, etc here - that we do in in `kube-setup-fence` - analogout to this code in `kube-setup-dashboard`:  https://github.com/uc-cdis/cloud-automation/blob/master/gen3/bin/kube-setup-dashboard.sh#L57 and https://github.com/uc-cdis/cloud-automation/blob/master/gen3/bin/kube-setup-dashboard.sh#L94

then either auto-add the bucket name to a new `dbgap-backup-g3auto` secret, or to `fence-config` - something like that.  What do you think?